t/errlog.t

```diff
@@ -1092,11 +1092,11 @@ GET /t
 --- response_body_like chomp
 \Alog lines: 2
 log level: 3
-log time: \d+\.\d+
+log time: \d+(\.\d+)?
```

Comment:
I think we should use `(?:)` instead of `()` to avoid useless submatch capturing?